doc/data/messages/i/import-private-name/details.rst

```diff
@@ -1 +0,0 @@
-You can help us make the doc better `by contributing <https://github.com/PyCQA/pylint/issues/5953>`_ !
```

Comment:
Using private import expose you to unexpected breaking changes in any version bump of your dependencies;
even patch versions.